profile/README.md

```diff
@@ -1,3 +1,6 @@
 <img src="https://github.com/mi6/.github/raw/main/profile/sis-gh-banner.png" alt="" />
 
 MI6 in open source follows a [Code of Conduct](https://github.com/mi6/ic-design-system/blob/main/CODE_OF_CONDUCT.md) and [Security Policy](https://github.com/mi6/ic-design-system/blob/main/SECURITY.md). Read the Design System's [Privacy Policy](https://design.sis.gov.uk/icds/privacy-policy).
+
+
+> Visit our websites to learn more about the departments: [MI6](https://www.sis.gov.uk) (The Secret Intelligence Service), [GCHQ](https://www.gchq.gov.uk) (The Government Communications Headquarters) and [MI5](https://www.mi5.gov.uk) (The Security Service).
```

Comment:
I think it might look better if it matches our ic-ui-kit readme
```suggestion
Visit our websites to learn more about:

- The [Secret Intelligence Service (MI6)](https://www.sis.gov.uk).
- The [Government Communications Headquarters (GCHQ)](https://www.gchq.gov.uk).
- The [Security Service (MI5)](https://www.mi5.gov.uk).
```